tests/tests_pytorch/trainer/connectors/test_checkpoint_connector.py

```diff
@@ -75,13 +76,6 @@ def test_hpc_restore_attempt(tmpdir):
         assert param.abs().sum() > 0
         torch.nn.init.constant_(param, 0)
 
-    # case 2: explicit resume path provided, restore hpc anyway
-    trainer = Trainer(default_root_dir=tmpdir, max_steps=3)
-    trainer.fit(model, ckpt_path="not existing")
```

Comment:
Before this got ignored, now, it would raise an error that "not existing" does not exist. Correct?

Reply:
That is correct. For clarity, I restored this test case with expecting annnnnnn exception